app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] As per requirements, please modify the association to belong to `author` with a class_name attribute set to `User`